proto/ibc/applications/fee/v1/genesis.proto

```diff
@@ -16,11 +16,14 @@ message GenesisState {
   // list of fee enabled channels
   repeated FeeEnabledChannel fee_enabled_channels = 2
       [(gogoproto.moretags) = "yaml:\"fee_enabled_channels\"", (gogoproto.nullable) = false];
+  // list of registered payees
+  repeated RegisteredPayee registered_payees = 3
+      [(gogoproto.moretags) = "yaml:\"registered_payees\"", (gogoproto.nullable) = false];
   // list of registered relayer addresses
-  repeated RegisteredRelayerAddress registered_relayers = 3
+  repeated RegisteredRelayerAddress registered_relayers = 4
```

Comment:
Why are there two of these? Shouldn't there be `RegisteredPayees` and `RegisteredCounterpartyPayees`?